PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -374,18 +379,19 @@
             return (config.sessionReplayConfig.maskAllImages && !isAsset) || view.isNoCapture()
         }
 
-        private func toWireframe(_ view: UIView) -> RRWireframe? {
+        private func toWireframe(_ view: UIView, isParentSensitive: Bool = false) -> RRWireframe? {
             if !view.isVisible() {
                 return nil
             }
 
             let wireframe = createBasicWireframe(view)
-
             let style = RRStyle()
+            var isSensitive = false
```

Comment:
I think this should be a local var within each if block otherwise a component that extends 2+ interfaces can swap the values, not sure if this will happen but on Android it'd, eg a Button is a `Button` and also a `TextView`, so if `isSensitive=true` and `isSensitive=false` in the 2nd case, you ignored the 1st result which is isSensitive=true